pkgs/development/python-modules/pyscf/default.nix

```
@@ -28,12 +31,57 @@ buildPythonPackage rec {
     numpy
     scipy
     h5py
+    cppe
   ];
 
+  checkInputs = [ nose nose-exclude ];
+
   PYSCF_INC_DIR="${libcint}:${libxc}:${xcfun}";
```

```suggestion
  PYSCF_INC_DIR="${libcint}:${libxc}:${xcfun}";
```
I think this should be a ``preBuild`` export?

---

Already in the `preConfigure` but yes :)